charts/cryostat/templates/auth_proxy_alpha_config.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: auth-proxy-alpha-config
+data:
+  upstreamConfig: |
+    server:
+      BindAddress: http://0.0.0.0:8080
+    upstreamConfig:
+      proxyRawPath: true
+      upstreams:
+        - id: cryostat
+          path: /
+          uri: http://cryostat:8181
+        - id: grafana
+          path: /grafana/
+          uri: http://grafana:3000
+        - id: storage
+          path: ^/storage/(.*)$
+          rewriteTarget: /$1
+          uri: http://s3:3000
+          passHostHeader: false
+          proxyWebSockets: false
+    providers:
+      - id: dummy
+        name: Unused - Sign In Below
+        clientId: CLIENT_ID
+        clientSecret: CLIENT_SECRET
+        provider: google
```

Comment:
Maybe we should take the same approach here that you used for the htpasswd file. Extract out this upstreamConfig YAML file into its own file and then use the `Files` template function with the `.oauth2Proxy.alphaConfig.confgFile` value, defaulting to the location of that YAML file.

Reply:
I  tried this, ended up not reading the file or file not found.